internal/user/api.go

```diff
@@ -1,13 +1,9 @@
 package user
```

Comment:
FFTI - Did you consider deleting this whole package? I would guess if we decided to add prefs we'd want to do something more like `ExtensionRepo`s.

Reply:
good point!